lib/pharos/config_schema.rb

```diff
@@ -116,6 +116,7 @@ def self.messages
             optional(:trusted_subnets).each(type?: String)
             optional(:known_peers).each(type?: String)
             optional(:password).filled(:str?)
+            optional(:ipalloc_cidr).filled(:str?)
```

Comment:
Just wondering if this is a good name... should it be `ipam_cidr` or something like that?

Reply:
This is for true power-users anyway, so I'd maybe use something that close to the actual Weave flag getting set so it's easy to refer to their docs too.

Reply:
`ipalloc_default_subnet:`?

Reply:
👍